src/index.ts

```diff
@@ -0,0 +1,10 @@
+import getSdk = require('resin-sdk');
+import settings = require('resin-settings-client');
+
+export = getSdk({
+	apiUrl: settings.get('apiUrl'),
+	imageMakerUrl: settings.get('imageMakerUrl'),
+	dataDirectory: settings.get('dataDirectory'),
+	apiVersion: 'v2',
```

Comment:
We tell the sdk what api version to use? Doesn't the SDK itself target a specific verson?